packages/theme/css/foundations/size.css

```diff
@@ -6,6 +6,8 @@
   --salt-size-icon: 12px;
   --salt-size-indicator: 1px;
   --salt-size-selectable: 12px;
+  --salt-size-bar-strong: 4px;
```

Comment:
is this a typo 'border-strong' instead of 'bar-strong' ?

Reply:
just crosschecked with excel sheet that's been verified by designs, it's indeed size-bar-strong